htdocs/compta/facture/list.php

```diff
@@ -1500,6 +1512,12 @@
 		print '<td class="liste_titre">';
 		print '</td>';
 	}
+	if (!empty($arrayfields['f.fk_fac_rec_source']['checked'])) {
+		// Template Invoice
+		print '<td class="liste_titre maxwidthonsmartphone right">';
+		$form->selectInvoiceRec($search_fac_rec_source, 'search_fac_rec_source');
```

Comment:
I think that using a combo here is not a good idea.
If combo is a complete list of element, it must be reserved to element that does not increase with time. But for invoice and recurring invoice, we may have more and more... Or we must allow a method that allow to fill the list after clicking on 1 or 2 keys (like we do for product, thirdparty, ...).

To avoid to develop this a better choice would be to simply offer an input field to allow to search on a string.

Reply:
Ok, done, Input text rather select select combobox